riff-raff/app/persistence/package.scala

```diff
@@ -1,12 +1,15 @@
 package persistence
 
+import akka.actor.FSM.->
 import magenta._
-import deployment.{PaginationView, DeployFilter}
+import deployment.{DeployFilter, PaginationView}
 import com.mongodb.casbah.commons.MongoDBObject
 import com.mongodb.casbah.Imports._
 import com.mongodb.DBObject
 import com.mongodb.casbah.MongoCursor
-import org.joda.time.{LocalDate, DateMidnight}
+import org.joda.time.{DateMidnight, LocalDate}
+
+import scala.collection.mutable
```

Comment:
I don't think this is needed as `+` creates a new set anyway.

Reply:
Ah yes! I originally added that when I was planning to mutate a Set and didn't remove it. Good spot.